tpg/tpg/sample.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Once we construct the library repo (further discussion of this will be made in it's own issue), this file won't need to exist.